.github/workflows/linting.yml

```
@@ -0,0 +1,19 @@
+name: Linting Salt-Toaster
+
+on: [push, pull_request]
+
+jobs:
+  tests:
+    runs-on: ubuntu-20.04
```

Just a small suggestion here:

```suggestion
    runs-on: ubuntu-latest
```

OTOH just wondering, is it possible to run this in openSUSE?

---

Sadly no. See: https://github.com/actions/virtual-environments#available-environments for the reason why they decided to go that way.